cdc_stream/start_command.cc

```diff
@@ -89,16 +109,33 @@ void StartCommand::RegisterCommandLineFlags(lyra::command& cmd) {
 absl::Status StartCommand::Run() {
   LogLevel level = Log::VerbosityToLogLevel(verbosity_);
   ScopedLog scoped_log(std::make_unique<ConsoleLog>(level));
-  LocalAssetsStreamManagerClient client(service_port_);
 
   std::string full_src_dir = path::GetFullPath(src_dir_);
   std::string user_host, mount_dir;
   RETURN_IF_ERROR(LocalAssetsStreamManagerClient::ParseUserHostDir(
       user_host_dir_, &user_host, &mount_dir));
 
+  LocalAssetsStreamManagerClient client(CreateChannel(service_port_));
   absl::Status status =
       client.StartSession(full_src_dir, user_host, ssh_port_, mount_dir,
                           ssh_command_, scp_command_);
+
+  if (absl::IsUnavailable(status)) {
+    LOG_DEBUG("StartSession status: %s", status.ToString());
```

Comment:
Maybe just have 1 output with LOG_INFO with status instead of two log entries.

Reply:
The idea behind it is that the user shouldn't see the unavail message (since it's pretty ugly and hard to understand), so I preferred having a nice message by default. However, in case unavail is reported unexpectedly, there's a way to debug it.